File: Accelbuffer/Accelbuffer/Enums/Endian.cs
namespace Accelbuffer
{
    /// <summary>
    /// 指示字节序
    /// </summary>
    public enum Endian : byte
    {
        /// <summary>
        /// 指示使用大端字节序（网络字节序）
        /// </summary>
        BigEndian = 0,

        /// <summary>
        /// 指示使用小端字节序
        /// </summary>
        LittleEndian = 1
    }
}

File: Accelbuffer/Accelbuffer.Java/src/accelbuffer/core/Endian.java
package accelbuffer.core;

/**
 *指示字节序
 */
public enum Endian {
    /**
     *指示使用大端字节序（网络字节序）
     */
    BIG_ENDIAN(0),
    /**
     *指示使用小端字节序
     */
    LITTLE_ENDIAN(1);

    private int value;

    private Endian(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }
}
